src/shared/install.c

```diff
@@ -873,7 +890,7 @@ static int find_symlinks(
                         continue;
                 }
 
-                r = find_symlinks_in_directory(d, path, root_dir, i, match_name, ignore_same_name, config_path, same_name_link);
+                r = find_symlinks_in_directory(d, path, root_dir, i, true, match_name, ignore_same_name, config_path, same_name_link);
```

Comment:
cryptic and long. maybe line beak? prefix the `true` with `/* ignore_destination= */`?